test/widgets/colors_checks.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:checks/checks.dart';
+import 'package:flutter_test/flutter_test.dart' as flutter_matcher;
+import 'package:flutter/painting.dart';
+import 'package:legacy_checks/legacy_checks.dart';
+
+extension ColorSwatchChecks<T> on Subject<ColorSwatch<T>> {
+  /// package:checks-style wrapper for [flutter_matcher.isSameColorSwatchAs].
+  void isSameColorSwatchAs(ColorSwatch<T> colorSwatch) {
+    legacyMatcher(flutter_matcher.isSameColorSwatchAs(colorSwatch));
+  }
+}
```

Comment:
Cool, I'm glad `legacy_checks` was useful!
I was worried I'd missed one for a second before I realized it was new. I know y'all follow the master channel, so, as I don't think I wrote down a flutter version support policy anywhere, I guess I'll just note here that I don't want to publish anything that unnecessarily requires a "nightly". I'm happy to accept PRs once they land in stable, but you'll probably still need to maintain a few of these.

Reply:
Sure, that approach makes sense.

Thanks to `legacyMatcher`, any individual wrappers like this one should be pretty cheap to write, like this one was.